packages/api/lib/testUtils.js

```diff
@@ -620,22 +617,6 @@ const createAsyncOperationTestRecords = async (context) => {
   };
 };
 
-const cleanupExecutionTestRecords = async (context, { arn }) => {
-  const {
-    knex,
-    executionPgModel,
-    esClient,
-    esIndex,
-  } = context;
-
-  await executionPgModel.delete(knex, { arn });
```

Comment:
Don't we need this?

Reply:
Hah, I mentioned in a previous comment that I did a search for the method `cleanupExecutionTestRecords` and did not find it being used anywhere. So I wondered if I should just remove it as I couldn't find it being used, and you confirmed...

Reply:
Ah ok, sure. Didn't piece together that this was the function we weren't sure was used.